tools/swc-transform-css-modules/package.json

```
@@ -0,0 +1,22 @@
+{
+  "private": "true",
+  "name": "@project-tools/swc-transform-css-modules",
+  "version": "0.1.0",
+  "author": {
+    "name": "Daniil Suvorov",
+    "email": "severecloud@gmail.com",
+    "url": "https://github.com/severecloud"
+  },
+  "main": "swc_transform_css_modules.wasm",
+  "scripts": {
+    "prepack": "yarn run build-wasi",
+    "build-wasi": "cargo build-wasi --release",
+    "postbuild-wasi": "cp target/wasm32-wasi/release/swc_transform_css_modules.wasm .",
+    "build": "echo use cargo build-wasi",
+    "test": "echo use cargo test",
+    "test:ci": "echo no test:ci"
+  },
+  "files": [
+    "swc_transform_css_modules.wasm"
+  ]
```

```suggestion
  "main": "build/swc_transform_css_modules.wasm",
  "scripts": {
    "prepack": "yarn run build-wasi",
    "build-wasi": "cargo build-wasi --release",
    "postbuild-wasi": "cp target/wasm32-wasi/release/swc_transform_css_modules.wasm build/.",
    "build": "echo use cargo build-wasi",
    "test": "echo use cargo test",
    "test:ci": "echo no test:ci"
  },
  "files": [
    "build/swc_transform_css_modules.wasm"
  ]
```

а то странно получается что прямо посреди всего...

или мб добавить `target/wasm32-wasi/release/swc_transform_css_modules.wasm` в ислючение в `.gitignore`?

---

Это норма

https://swc.rs/docs/plugin/publishing#creating-a-npm-package-for-plugin
https://github.com/swc-project/plugins/blob/main/packages/noop/package.json#L7

---

Выглядит не очень, конечно)